gui/widgets/qopenglmousetracker.cpp

```diff
@@ -20,6 +20,66 @@ QOpenGLMouseTracker::QOpenGLMouseTracker(QWidget *parent, float lineWidth,
       m_backgroundColor(QWidget::palette().color(QWidget::backgroundRole())),
       m_lastColour(QWidget::palette().color(QWidget::backgroundRole()).rgb()) {}
 
+std::shared_ptr<rendering::ClippingPlane>
+QOpenGLMouseTracker::BuildClippingPlane(GLfloat a, GLfloat b, GLfloat c,
```

Comment:
There are two QOpenGLMouseTracker::BuildClippingPlane functions that have the same contents except for one line, and the first one isn't tested - if not needed could you remove the unused code?
Or if you need both then could you refactor them to remove the copy&pasted logic?

Reply:
Both of them might be of use and are rapping other 2 functions that can be used for updating the planes. I don't have a nice way of getting away from the redundancy without adding extra functions that will also be unuseful for other than this case.

Reply:
OK, then I guess let's merge this as is and then see what functions end up being used in the full implementation